build.gradle

```diff
@@ -11,6 +12,7 @@ buildscript {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$versions.kotlin"
 
         classpath "com.novoda:bintray-release:$versions.bintrayPlugin"
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
```

Comment:
Why is did you add this line? isn't it the same as line 12?

Reply:
😅